control-base/src/main/java/fi/nls/oskari/control/data/GetPrintHandler.java

```diff
@@ -118,19 +120,44 @@ private PrintRequest createPrintRequest(ActionParameters params)
         request.setShowLogo(params.getHttpParam(PARM_LOGO, false));
         request.setShowScale(params.getHttpParam(PARM_SCALE, false));
         request.setShowDate(params.getHttpParam(PARM_DATE, false));
-
+        request.setScaleText(params.getHttpParam(PARM_SCALE_TEXT, ""));
         setPagesize(params, request);
         setCoordinates(params.getRequiredParam(PARM_COORD), request);
         setFormat(params.getRequiredParam(PARM_FORMAT), request);
+        request.setUseConfiguredScale(params.getHttpParam(PARM_USE_CONFIGURED_SCALE, false));
```

Comment:
I don't see a reason for using two parameters to trigger the UseConfiguredScale-mode.